File: utilities/src/main/java/com/toocol/ssh/utilities/log/Logger.java
package com.toocol.ssh.utilities.log;

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/6/28 11:55
 */
public interface Logger {
    void debug(String message, Object... params);

    void info(String message, Object... params);

    void warn(String message, Object... params);

    void error(String message, Object... params);
}

File: utilities/src/main/java/com/toocol/termio/utilities/log/Logger.kt
package com.toocol.termio.utilities.log

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/6/28 11:55
 */
interface Logger {
    fun debug(className: String, message: String?, vararg params: Any?)

    fun info(className: String, message: String?, vararg params: Any?)

    fun warn(className: String, message: String?, vararg params: Any?)

    fun error(className: String, message: String?, vararg params: Any?)
}